metadata/move.xml

```diff
@@ -1,31 +1,32 @@
 <?xml version="1.0"?>
 <wayfire>
 	<plugin name="move">
-	<_short>Move Window</_short>
-	<category>Window Management</category>
-	<option name="activate" type="button">
-		<_short>Activate</_short>
-		<default>&lt;super&gt; BTN_LEFT</default>
-	</option>
-	<option name="enable_snap" type="bool">
-		<_short>Snap</_short>
-		<default>true</default>
-	</option>
-	<option name="snap_threshold" type="int">
-		<_short>Snap Threshold</_short>
-		<default>2</default>
-	</option>
-	<option name="enable_snap_off" type="bool">
-		<_short>Snap Off</_short>
-		<default>true</default>
-	</option>
-	<option name="snap_off_threshold" type="int">
-		<_short>Snap Off Threshold</_short>
-		<default>0</default>
-	</option>
-  <option name="join_views" type="bool">
-    <_short>Disallow independently moving dialogues</_short>
-    <default>false</default>
-  </option>
+		<_short>Move</_short>
+		<category>Window management</category>
+		<option name="activate" type="button">
+			<_short>Activate</_short>
+			<default></default>
```

Comment:
Should be "none" instead of empty. Same goes for the other places where you disable the bindings by default.